src/cultionet/augment/augmenter_utils.py

```diff
@@ -0,0 +1,244 @@
+import typing as T
```

Comment:
This file is a copy/paste from the deleted `augmentation.py` for reorganization purposes.